nss-day-cohort-47/tabloidfullstack-eureka
# back end is listing comments based on a post id

# Description
Added back end functionality so that the list of comments displays when a certain post id is typed.
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Run visual studio and open the get comments section on swagger. Type in any post id and hit execute to see the list of comments.
# Checklist:
- [] I have commented my code, particularly in hard-to-understand areas
